Bundle/CoreBundle/Resources/doc/SETUP.md

```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
 ```
 
 - Enable the `localizeddate` filter:
+# Fresh install including base config and dependencies
+
+## symfony 2.8
+
+```sh
+mkdir -p ~/workspace/clientName/projectName && cd ~/workspace/clientName/projectName
+symfony new . 2.8
```

Comment:
I prefer to use Composer personally, because I already have Composer and I don't want to use Symfony's own installer: http://symfony.com/doc/current/setup.html#creating-symfony-applications-with-composer

What do you think?

Or we can add a link to http://symfony.com/doc/current/setup.html and let users follow the Symfony official guide.